python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/system.schema.yml

```diff
@@ -24,24 +24,28 @@ keys:
                 description: Segment size.
           ipv4_access_groups:
             type: list
-            primary_key: acl_name
+            unique_keys:
+              - vrf
```

Comment:
I believe you can just set `unique: true` under the key itself. That will show up properly in docs.

Reply:
It was giving error while building schema using pre-commit.
Also, I couldn't find `unique` key in https://github.com/aristanetworks/avd/blob/devel/python-avd/pyavd/_schema/avd_meta_schema.json

Reply:
ok I am mixing with some other schema work.